systemtests/battery_test.go

```diff
@@ -2,78 +2,93 @@ package systemtests
 
 import (
 	"fmt"
+	. "gopkg.in/check.v1"
```

Comment:
Please put this line, surrounded by whitespace lines, on line 11. This is where https://github.com/bradfitz/goimports would put it by default (we can discuss tooling further offline if you want)